.gitlab-ci.yml

```diff
@@ -152,15 +152,14 @@ build_preview:
     - touch preview_error.log # Create log file for errors
     - exec 2> $PREVIEW_ERROR_LOG # We want to output any detected error to a log for use in notifications
     - make dependencies
-    - make vector_data
```

Comment:
@davidejones do we want to remove the related `vector_data` rule and `/assets/scripts/reference-process.js` script?

Reply:
not just yet, that code is still needed and this is running on master still so just removing on preview